fields/types/html/HtmlField.js

```diff
@@ -204,10 +204,12 @@ module.exports = Field.create({
 	},
 
 	renderValue () {
+		const htmlValue = {
+			__html: this.props.value
```

Comment:
@AurelioDeRosa The [build is failing](https://travis-ci.org/keystonejs/keystone/jobs/184122696#L508) due to a linting error here - you're missing the trailing comma.

You can check for lint errors by running `npm run lint`, and probably fix it by running `npm run lint-fix`. I thought this was included in `npm run test-all`, but I guess not.